brave-tests/pom.xml

```diff
@@ -44,6 +44,10 @@
       <groupId>org.assertj</groupId>
       <artifactId>assertj-core</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.awaitility</groupId>
```

Comment:
I really don't like adding this as a new dep everyone depends on this now needs to pull in. I'm aware you are a fan of this, but personally I'm not.

Reply:
another option is to see if this is needed in the main deps. if this can be moved to test scope, ok by me.